src/test/resources/application.properties

```diff
@@ -1 +1,3 @@
 application.staging.url=${STAGING_URL:http://localhost:8080}
+api.user.username=${ACTIONS_TEST_USERNAME}
```

Comment:
Why did you call the env variable like that and not just `API_USER_USERNAME`?

Reply:
it was also my thought, but then this env var is required during build, so I name it with easier to think about `ACTIONS_TEST_USERNAME` means that this env will be available only in Github Actions and the setting is available in Github, but I can rename it as well to reflect what is currently set in our infra

Reply:
I see your point. Maybe then just add a default value, since this is only used for tests?